index.js

```diff
@@ -68,7 +68,7 @@ function buildAPI(loginOptions, html, jar) {
 
   var apiFuncNames = [
     'listen',
-    'getUserID',
+    'getUserId',
```

Comment:
Can you change that back to `ID`? We're trying to make everything consistent and decided on writing it `ID` instead of any other way.